charts/sbombastic/tests/controller/deployment_test.yaml

```diff
@@ -24,3 +24,17 @@ tests:
       - contains:
           path: "spec.template.spec.containers[0].args"
           content: "-log-level=debug"
+  - it: "should render a Deployment with the default values"
+    asserts:
+      - equal:
+          path: "spec.replicas"
+          value: 3
+      - equal:
+          path: "spec.template.spec.containers[0].image"
+          value: "ghcr.io/rancher-sandbox/sbombastic/controller:0.1.0-alpha1"
+      - equal:
+          path: "spec.template.spec.containers[0].imagePullPolicy"
+          value: "IfNotPresent"
+      - contains:
+          path: "spec.template.spec.containers[0].args"
+          content: "-log-level=info"
```

Comment:
I think we can remove this test, it would break in the future once we update the `AppVersion` because the tag would change.

I would rather put the assertion about `ghcr.io` being present inside of the previous test